src/txdb.cpp

```diff
@@ -697,12 +697,7 @@ uint64_t GetTotalSystemMemory()
 }
 #endif
 
-void GetCacheConfiguration(int64_t &_nBlockDBCache,
-    int64_t &_nBlockUndoDBcache,
-    int64_t &_nBlockTreeDBCache,
-    int64_t &_nCoinDBCache,
-    int64_t &_nTxIndexCache,
-    bool fDefault)
+CacheConfig SetCacheConfiguration(bool fDefault)
```

Comment:
Switching from "Get" to "Set" here is showing how this naming is not topically right.  How about "Calculate", "Derive", or "Discover" -- e.g. "DiscoverCacheConfiguration"?

Also, please add doc comment

Reply:
done